### PR TITLE
Shapes in routes

### PR DESCRIPTION
**Pull request by antoine-de**
output shapes as geojson in api:
- lines
- routes
- route_schedule

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/fields.py [hunk 1]
```diff
@@ -273,41 +273,51 @@ def output(self, key, obj):
         return res
 
 
-class GeoJson(fields.Raw):
+class Geojson(fields.Raw):
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/interfaces/v1/fields.py]: `class MultiLineString`?

File: source/jormungandr/tests/ptref_tests.py [hunk 2]
```diff
@@ -83,3 +83,32 @@ def test_vj_depth_3(self):
...
+        routes = get_not_null(response, 'routes')
+
+        for r in routes:
+            is_valid_route(r, depth_check=1)
+
+            #we know we have a geojson for this test so we can check it
+            geo = get_not_null(r, 'geojson')
+            shape(geo) #TODO check length
+            assert geo['properties'][0]['length'] != 0
+            #TODO check length size
```
Comment (TeXitoi) [inline source/jormungandr/tests/ptref_tests.py]: lack a `\n` at the end of the file

File: source/type/type.proto [hunk 3]
```diff
@@ -138,6 +138,15 @@ enum OdtLevel{
 	zonal = 2;
 	all = 3;
 }
+
+message Coords {
+    repeated GeographicalCoord coordinates = 1;
+}
+
+message GeoJson {
```
Comment (TeXitoi) [inline source/type/type.proto]: MultiLineString?

File: source/type/type.proto [hunk 4]
```diff
@@ -138,6 +138,15 @@ enum OdtLevel{
 	zonal = 2;
 	all = 3;
 }
+
+message Coords {
```
Comment (TeXitoi) [inline source/type/type.proto]: LineString?

**Comment by antoine-de** [comment]
Jenkins, test this please

**Pull request merged**